crossbeam-epoch/Cargo.toml

```diff
@@ -41,6 +41,12 @@ cfg-if = "1"
 const_fn = "0.4"
 memoffset = "0.6"
 
+# An internal configuration option used when running loom.
+# This is not part of the public API and crossbeam may make breaking changes
```

Comment:
> This is not part of the public API

I thought you were trying to make this an unstable public API used by the downstream crate (https://github.com/crossbeam-rs/crossbeam/issues/486#issuecomment-610977956).
If so,  "An internal configuration" and "not part of the public API"  are not a proper description, and perhaps something like "An unstable configuration" and "not part of the stable public API" preferred.